src/components/Fritekst.tsx

```diff
@@ -0,0 +1,26 @@
+import { Textarea, TextareaProps } from "nav-frontend-skjema";
+import React from "react";
+
+export type FritekstSize = "medium" | "stor" | "ekstra-stor";
+type FritekstProps = {
+  size: FritekstSize;
+} & TextareaProps;
+
+const Fritekst = ({ size, ...rest }: FritekstProps) => (
+  <Textarea {...rest} textareaClass={textAreaClass(size)} />
+);
+
+const textAreaClass = (size: FritekstSize) => {
+  switch (size) {
+    case "medium":
+      return "fritekstMedium";
+    case "stor":
+      return "fritekstStor";
+    case "ekstra-stor":
+      return "fritekstEkstraStor";
+    default:
+      return undefined;
+  }
+};
```

Comment:
Hvorfor egendefinerte størrelser som man mapper om, i stedet for å bruke riktig navn til å begynne med?

Reply:
Hm, godt spørsmål 😅 Tanken var nok å ikke bruke så generiske css-klassenavn som "medium" osv, men kan godt endre til å bruke f.eks. "fritekst-medium" osv over hele linja.

Reply:
Det er ikke veldig krise for min del. Det er kanskje bedre å ha enkle, generiske navn der dette brukes, og så mappe om, i stedet for å ha vanskeligere navn men slippe switch 🤷

Reply:
Ja, tror jeg lander på at det er best sånn det er 😃